organization/welcome-email.md

```diff
@@ -0,0 +1,29 @@
+Hello,
+
+We are excited for the first Building the Peer-to-Peer Internet workshop, which starts Tuesday April 10 and running weekly on Tuesdays until May 15, and hope you are too! A few notes:
+
+1. Be sure to complete the pre-workshop survey [ link to survey ]. We use this to calibrate the pace of the workshop and, together with a post-workshop survey, to assess how it went in order to improve.
+2. Make sure you have administrator privileges on your laptop. If you are interested, you can install the Shell and Text Editor software we will be using for the workshop (for many of you it will be already installed!): [Setup Instructions](https://github.com/tomeshnet/p2p-internet-workshop/blob/master/SETUP.md). If you have any trouble, send us an email at hello@tomesh.net or come a little early Tuesday April 10 before the class.
+
+If you are no longer able to attend, please let us know at hello@tomesh.net so that we can offer your seat to someone on the waitlist.
+
+Links that will be used in the workshop: 
+- workshop page https://tomesh.net/peer-to-peer-internet/ 
+- lesson materials https://github.com/tomeshnet/p2p-internet-workshop/
+- shared notepad https://pad.riseup.net/p/peer-to-peer-internet
+
+You may also want to check out these resources about the Unix Shell:
+- Learnshell.org https://www.learnshell.org/
+- Software Carpentry's [Introducing the Shell](https://swcarpentry.github.io/shell-novice/01-intro/) and [Navigating Files and Directories](https://swcarpentry.github.io/shell-novice/02-filedir/)
```

Comment:
Link syntax here but not above

Maybe its overkill. Perhaps https://linuxjourney.com/lesson/the-shell and tell them to do like the first 6-7 lessons (unless they want to do more)